packages/react-core/src/components/Button/Button.tsx

```diff
@@ -58,6 +64,8 @@ export interface ButtonProps extends Omit<React.HTMLProps<HTMLButtonElement>, 'r
   inoperableEvents?: string[];
   /** Adds inline styling to a link button */
   isInline?: boolean;
+  /** Adds small or large styling to the button */
+  size?: 'sm' | 'md' | 'lg';
```

Comment:
Including here `md` is a cool idea! However the only issue with this is that we don't actually have a modifier for the "md" variant (since we use that size as the default), so nothing gets set on `pf-c-button` even though in different circumstances we might expect to see something like `pf-m-medium` when `size="md"`. 

[Accordion only supports `default` and `lg`](https://github.com/patternfly/patternfly-react/blob/4d73b9030cf92b075c46a0eeab561d47070bec50/packages/react-core/src/components/Accordion/Accordion.tsx#L20), so following that convention we probably should use `default` here instead of `md`... @mcoker what are your thoughts?

```suggestion
  size?: 'default' | 'sm' | 'lg';
```

Reply:
i just used `md` based on spinner's size options - but open to whatever